src/k9policy.ts

```diff
@@ -61,6 +61,44 @@ export interface IAWSServiceAccessGenerator {
   makeConditionsToExceptFromDenyEveryoneElse(): Conditions;
 }
 
+/**
+ * Check whether the provided access specs ensure that at least one principal can both read and administer configuration.
+ * @param accessSpecsByCapability is a map of access specs keyed by access capability
+ *
+ * @return true when at least one principal that can administer and read configuration exists
+ */
+export function canPrincipalsManageResources(accessSpecsByCapability: Map<AccessCapability, IAccessSpec>) {
+  console.log(`canPrincipalsManageResources eval'ing ${accessSpecsByCapability}`);
```

Comment:
Console logging in a CDK library is a bit unusual. I'd advise debug here. 

I forgot to mention that a while back. I swallow your console.log's from the KMS key policy generation to hide them. :)

Reply:
sorry, that's not supposed to be there. will remove.